src/js/components/Layer/LayerContainer.js

```diff
@@ -51,6 +60,9 @@ class LayerContainer extends Component {
     const { position } = this.props;
     if (position !== 'hidden') {
       this.makeLayerVisible();
+      if (this.anchorRef.current) {
+        this.anchorRef.current.focus();
```

Comment:
Does this prevent the caller from also trying to set focus on a particular input element in the Layer?

Reply:
I dont think so since componentDidMount on layercontainer will always be called first.

Reply:
Can you at least add a comment explaining why we add this anchor and give it focus?